common/script/ops/webhooks.js

```diff
@@ -0,0 +1,152 @@
+import refPush from '../libs/refPush';
+import validator from 'validator';
+import i18n from '../i18n';
+import { v4 as uuid } from 'uuid';
+import {
+  BadRequest,
+  NotFound,
+} from '../libs/errors';
+
+const TASK_ACTIVITY_DEFAULT_OPTIONS = Object.freeze({
```

Comment:
we're not doing this anywhere else, it's okay for me if we decide to use it for config objects but it should be standard.